report-viewer/src/store/store.ts

```diff
@@ -0,0 +1,62 @@
+import { createStore} from "vuex";
+
+/**
+ * Local store. Stores the state of the application.
+ */
+const store = createStore({
+    state () {
+        return {
+            /**
+             * The set of ids to be hidden.
+             */
+            anonymous : new Set(),
+            /**
+             * Stored files if zip mode is used. Stores the files as key - file name, value - file string
+             */
+            files : Object,
+            /**
+             * Indicates whether local mode is used.
+             */
+            local: Boolean,
+            /**
+             * Indicates whether zip mode is used.
+             */
+            zip: Boolean,
+            /**
+             * Indicates whether single file mode is used.
+             */
+            single: Boolean,
+            /**
+             * Files string if single mode is used.
+             */
+            fileString: String
```

Comment:
I dont't know whether vuex supports optional parameters, but if this is the case, they could be used (also) here